utils/checks.stanza

```diff
@@ -47,20 +47,36 @@ Set of standard checks to run on a top-level-module.
 <S>
 
 public defn check-design (module:JITXObject):
+  erc-check-design(module)
+  drc-check-design(module)
+
+public defn erc-check-design (module:JITXObject):
   inside pcb-module:
     for i in populated-components(module) do :
       check rated-temperature(i)
-      check capacitor-component(i) when has-property?(i.capacitance)
-      check resistor-component(i)  when has-property?(i.resistance)
-      check inductor-component(i)  when has-property?(i.inductance)
+      check capacitor-component(i) when has-property?(i.capacitor)
+      check resistor-component(i)  when has-property?(i.resistor)
+      check inductor-component(i)  when has-property?(i.inductor)
+      check diode-component(i)     when has-property?(i.diode)
```

Comment:
Note: we should make sure that importers that attach these properties follow the new convention so checks are run on imported designs

Reply:
I looked in the generic component code and we do annotate the properties, but we should also check the importer code assuming we know the type of component that we are importing.